packages/inter-protocol/src/reserve/assetReserve.js

```diff
@@ -95,17 +133,12 @@ const start = async (zcf, privateArgs) => {
     params.getAmmInstance(),
   );
 
-  /** @type {ZCFMint} */
-  const runMint = await zcf.registerFeeMint(RunKW, privateArgs.feeMintAccess);
-  const runKit = runMint.getIssuerRecord();
-  saveBrandKeyword(runKit.brand, RunKW);
-  // no need to saveIssuer() b/c registerFeeMint does it
-
   /**
+   * @param {MethodContext} _context
    * @param {Issuer} issuer
    * @param {string} keyword
    */
-  const addIssuer = async (issuer, keyword) => {
+  const addIssuer = async (_context, issuer, keyword) => {
```

Comment:
we don't have a style for this yet but my 2¢ would be to leave it `context` so it git blame doesn't churn when you want to start or stop reading from it in the body. There's a reason the lint rules don't complain about it.

Reply:
I thought our lint rules do complain about unused params that don't begin with underbar. ??

Reply:
Not in param lists.  I figure because you necessarily have to bind earlier params to receive the latter ones.
<img width="659" alt="Screen Shot 2022-07-25 at 10 12 43 AM" src="https://user-images.githubusercontent.com/21505/180835897-a593779c-32da-4548-8596-a8d805318080.png">

Reply:
I think I prefer to keep the underbar for unused variables in general.